fastlane/Fastfile

```diff
@@ -861,7 +878,7 @@ class Build
             height = Integer(image_height) / 4.0
 
             # Add build number to the icon
-            %x( convert -background '#0008' -fill white -gravity center -size #{width}x#{height} caption:"#{extra_info}" "#{image}" +swap -gravity south -composite "#{image}" )
+            %x( magick -background '#0008' -fill white -gravity center -size #{width}x#{height} caption:"#{extra_info}" "#{image}" +swap -gravity south -composite "#{image}" )
```

Comment:
fixes the warning:
`WARNING: The convert command is deprecated in IMv7, use "magick" instead of "convert" or "magick convert"`